scripts/actions/spells/trust/nashmeira_ii.lua

```diff
@@ -12,7 +12,38 @@ spellObject.onSpellCast = function(caster, target, spell)
 end
 
 spellObject.onMobSpawn = function(mob)
-    xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
+    xi.trust.teamworkMessage(mob, {
+        [xi.magic.spell.LILISETTE_II] = xi.trust.messageOffset.TEAMWORK_1,
+        [xi.magic.spell.ARCIELA_II] = xi.trust.messageOffset.TEAMWORK_2,
+        [xi.magic.spell.IROHA_II] = xi.trust.messageOffset.TEAMWORK_3,
+        [xi.magic.spell.LION_II] = xi.trust.messageOffset.TEAMWORK_4,
+        [xi.magic.spell.PRISHE_II] = xi.trust.messageOffset.TEAMWORK_5,
+    })
+
+    mob:addSimpleGambit(ai.t.PARTY, ai.c.HPP_LT, 75, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.CURE)
+    -- TODO: Should only use Curaga when *3* or more party members are below 75%
+    -- Setting the Curaga threshold a bit lower here to prevent a race condition with regular Cure gambit, above.
```

Comment:
Thats not a race condition if it were you would not be able to avoid it with this. What you have can be thought of as a logical or priority conflict. The 2 aren't racing and 2 threads seeing who gets there 1st.

Reply:
I've updated my comment to more accurately reflect my intent.